index.html

```diff
@@ -120,179 +120,321 @@ <h3 class="project-title">Multi-Post stories</h3>
+        you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <a class="res" href="#"><img src="images/resume.png" alt="resume" /></a>
 
-        <img class= "about-ill" src="images/illustration/illustration-about-me-mobile.svg" alt="about">
+      <img
+        class="about-ill"
+        src="images/illustration/illustration-about-me-mobile.svg"
+        alt="about"
+      />
 
-        <div class="border">
-          <img src="images/Vector4.png" alt="line">
-        </div>
+      <div class="border">
+        <img src="images/Vector4.png" alt="line" />
+      </div>
 
-        <div class="language">
-          <img src="images/Rectangle55.png" alt="rectangle">
-          <h2 class="lang-header">Language</h2>
-          
-        </div>
-        <div class="skills">
-          <ul>
-            <li>Javascript</li>
-            <li>Ruby</li>
-            <li>Html</li>
-            <li>Css</li>
-          </ul>
-        </div>
-        <div class="language">
-          <img src="images/Rectangle56.png" alt="rectangle">
-          <h2 class="lang-header">Frameworks</h2>
-          
-        </div>
-        <div class="skills">
-          <ul>
-            <li>Bootstrap</li>
-            <li>Ruby on Rails</li>
-            <li>RSPec</li>
-            <li>CapyBara</li>
-            <li>Selenium</li>
-          </ul>
-        </div>
-        <div class="language">
-          <img src="images/Rectangle57.png" alt="rectangle">
-          <h2 class="lang-header">Skills</h2>
-          
-        </div>
-        <div class="skills">
+      <div class="language">
+        <img src="images/Rectangle55.png" alt="rectangle" />
+        <h2 class="lang-header">Language</h2>
+      </div>
+      <div class="skills">
+        <ul>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+      </div>
+      <div class="language">
+        <img src="images/Rectangle56.png" alt="rectangle" />
+        <h2 class="lang-header">Frameworks</h2>
+      </div>
+      <div class="skills">
+        <ul>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+      <div class="language">
+        <img src="images/Rectangle57.png" alt="rectangle" />
+        <h2 class="lang-header">Skills</h2>
+      </div>
+      <div class="skills">
+        <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </section>
+
+    <footer class="contact-section">
+      <div class="contact">
+        <h3>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+
+        <form action="https://formspree.io/f/mpzbdpaw" method="post">
```

Comment:
Good job on implementing the design for contact section but please use [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) to guide your design should look. You have a few issues on design that you have to fix them and you are done. 💯 
Here you have a screenshot of how I am seeing it:

![image](https://user-images.githubusercontent.com/74506933/154077911-94f56894-538a-464b-86a6-c4a795d2b295.png)


Here is contact section on Figma design:
![image](https://user-images.githubusercontent.com/74506933/154077749-06c8404a-bf46-474e-b229-63a158dcba38.png)

Reply:
I have made the changes on the figma template